firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Ranges.txt

```diff
@@ -0,0 +1,100 @@
+Scenario:The Ranges:  clamp and within function works correctly
+  GIVEN('The clamp and within function')
+//floor>value>ceiling
```

Comment:
What I see in your WHENs for this GIVEN is that mathematically `floor < value` and `value < ceiling`, i.e. `floor < value < ceiling`, not `floor > value > ceiling`. Other comments in this file need to be corrected in this way, too.